storage/integration_test.go

```diff
@@ -4832,3 +4835,16 @@ func skipGRPC(reason string) context.Context {
 func skipHTTP(reason string) context.Context {
 	return context.WithValue(context.Background(), skipTransportTestKey("http"), reason)
 }
+
+// Extract the error code if it's a googleapi.Error
+func extractErrCode(err error) int {
+	if err == nil {
+		return 0
```

Comment:
Nit: perhaps `http.StatusOK`?

Reply:
This was extracted from a method (see my requester pays PR).  I could change it, but I'm not sure it's more accurate, as we can't check if the error *would be* a googleapi.Error if it existed. For example, in the RequesterPays test I pass in an error that may be a gRPC error - it might be confusing in that case for this method to return a 200?

Reply:
That's fair! I'm OK with that as well. I trust your judgement.